packages/ui/src/apps/collectionsUi/viewModules/start/Component.js

```diff
@@ -1,126 +1,128 @@
+            style={{ marginTop: '6.25em', maxWidth: '60em' }}
+            verticalAlign="middle"
+          >
+            <Grid.Column width={10}>
+              <Header
+                as="h1"
+                content={`${moduleTranslate({
+                  capitalize: true,
+                  module: 'start',
+                  textKey: 'welcome',
+                })} ${capitalizeFirstLetterOfEachWord(
+                  username
+                )}! ${moduleTranslate({
+                  capitalize: true,
+                  module: 'start',
+                  textKey: 'startPageText',
+                })}`}
+                style={{
+                  color: '#000000a0',
+                  fontWeight: 200,
+                  lineHeight: '2.5rem',
+                }}
+              />
+              <div
+                style={{
+                  marginBottom: '1em',
+                  marginTop: '2em',
+                }}
+              >
+                <NavLink to="/app/specimens/mammals/create/sections/0">
+                  <Button primary size="huge" type="button">
+                    <Icon name="plus" />
+                    {moduleTranslate({
+                      capitalize: true,
+                      module: 'start',
+                      textKey: 'btnRegisterMammal',
+                    })}
+                  </Button>
+                </NavLink>
+                <NavLink to="/app/specimens/mammals">
+                  <Button basic color="blue" size="huge" type="button">
+                    <Icon name="search" />
+                    {moduleTranslate({
+                      capitalize: true,
+                      module: 'start',
+                      textKey: 'btnFindMammals',
+                    })}
+                  </Button>
+                </NavLink>
+                <br />
+              </div>
+              <div
+                style={{
+                  color: '#000000a0',
+                }}
+              >
+                {moduleTranslate({
+                  capitalize: true,
+                  module: 'start',
+                  textKey: 'needHelp',
+                })}{' '}
+                <a href="mailto:team@mail.dina-web.net">
+                  dina-collections@nrm.se
+                </a>
+              </div>
+            </Grid.Column>
+            <Grid.Column width={6}>
+              <Image centered src={logo} style={{ height: '16rem' }} />
+            </Grid.Column>
           </Grid>
-        </Segment>
+        </div>
+        <Divider style={{ marginTop: '5em' }} />
+        <div style={{ color: '#000000a0', paddingLeft: '1em' }}>
+          {moduleTranslate({
+            capitalize: true,
+            module: 'start',
+            textKey: 'dinaCollections',
+          })}{' '}
+          <i>
+            {moduleTranslate({
+              capitalize: true,
+              module: 'start',
+              textKey: 'currentlyVersion',
+            })}
+            1.0
```

Comment:
we're not on version 1.0. if you look in package.json in the root we're  "0.5.2" now. what is in the sketch is not always meant literally, here it means to show the current version, not hardcode 1.0 :-)

you will have to get the version from package.json somehow... and that is not trivial, because create-react-app blocks import of anything outside `/src` unless it's the ui package.json or something symlinked. so we probably need to add a function in common that imports the root package.json and returns the version. maybe I need to help with that

some info in here: https://github.com/facebook/create-react-app/pull/2468
https://stackoverflow.com/questions/44114436/the-create-react-app-imports-restriction-outside-of-src-directory